src/lib/array.jl

```diff
@@ -729,6 +715,8 @@ end
   end
 end
 
+# ChainRules has this also but does not use FillArrays, so we have out own defination
```

Comment:
defination -> definition

Reply:
you can use the "suggestion" feature
![image](https://user-images.githubusercontent.com/13448787/80982320-1412d700-8e23-11ea-940f-213808e35311.png)

Reply:
Sorry, I did not realize I can do that – GitHub's UI keeps getting more complicated. Will do so going forward, thanks.